src/player-service/playerRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { getPlayerById, getPlayersByTeam, placePlayerOnTransferList, transferPlayer } from './playerController';
+import { authMiddleware } from '../libs/common/middleware/auth';
+
+const router = Router();
+
+router.use(authMiddleware);
```

Comment:
How can the user create players?

Reply:
the user isnt able to create players, since those are randomly generated, just transfer(buy and sell) but tht isnt fully implemented yet